src/core/hle/service/nvflinger/nvflinger.cpp

```diff
@@ -139,11 +139,15 @@ std::optional<u64> NVFlinger::CreateLayer(u64 display_id) {
     }
 
     const u64 layer_id = next_layer_id++;
+    CreateLayerAtId(display, layer_id);
+    return layer_id;
+}
+
+void NVFlinger::CreateLayerAtId(VI::Display* display, u64 layer_id) {
```

Comment:
Perhaps use a reference like so:

```suggestion
void NVFlinger::CreateLayerAtId(VI::Display& display, u64 layer_id) {
```

Since the only way the intended behavior of this function can work is if the display isn't null.